update/release.go

```
@@ -196,7 +196,7 @@ func (s ReleaseSpec) calculateImageUpdates(rc ReleaseContext, candidates []*Serv
 
 	switch s.ImageSpec {
 	case ImageSpecLatest:
-		images, err = collectUpdateImages(rc.Registry(), candidates)
+		images, err = collectUpdateImages(rc.Registry(), candidates, logger)
```

Are you sure we want to fail silently in this case?